drltrace/scripts/headers_parser.py

```diff
@@ -30,32 +29,36 @@
 # DAMAGE.
 #
 # ********************************************************************************
-# The script is used to search for WinAPI function prototypes in the SDK's headers
+# The script is used to search for WinAPI function prototypes in the SDK's headers based
+# on the list/lists of exported functions and to save them in a separate set of files for
+# the further postprocessing using gen_drltrace_config.py.
+#
+# The script saves possible function prototypes using the following pattern:
+# [exported function name] -> [possible return types] [function name found]([args])
+# where [exported function name] is a name of function we are searching for.
 
 import os
 import sys
 from os import listdir
 from os.path import isfile, join
+import argparse
 
-# the list of ignored expored functions
+# the list of ignored exported functions
 common_dll_names = {"DllCanUnloadNow", "DllGetClassObject", "DllRegisterServer",
                     "DllUnregisterServer", "DllEntryPoint", "DllInstall"}
 
-# the path where a list or lists of WinAPI functions to search for are located
-EXPORT_PATH = "exports\\"
-
-# the path where Windows headers are located (install Microsoft SDK to have them)
-HEADERS_PATH = "C:\\Program Files\\Microsoft SDKs\\Windows\\v7.1\\Include\\"
-
-# the path where the script saves results
-RESULTS_PATH = "results\\"
+# the maximum number of elements in a function prototype
+PROTOTYPE_LENGTH = 70
```

Comment:
This is again mixing lower-case and all-caps variable names: `common_dll_names`, `PROTOTYPE_LENGTH`.   Are you following the style guide for all-caps for constants?  Then they should *both* be all-caps.